core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -233,7 +233,7 @@ mainView.balance.locked.short=Locked
 mainView.footer.usingTor=(using Tor)
 mainView.footer.localhostBitcoinNode=(localhost)
 mainView.footer.btcInfo=Bitcoin network peers: {0} / {1} {2}
-mainView.footer.btcInfo.initializing=Initializing
+mainView.footer.btcInfo.initializing=Initializing Bitcoin network connection
```

Comment:
Why not "Connecting to Bitcoin network"

Reply:
I'll remove the Initializing DAO completely and will change the old Initializing to "Connecting to Bitcoin network"